### ISSUE TITLE
Running applications

### ISSUE THREAD

**Issue by davbotjohnson** [george-hopkins/opentom#33]
For some reason only some of my applications will work, would this be caused by insufficient ram? Thanks!

**Comment by george-hopkins** [comment]
Yes, that might be the case. Do they work if you start them individually?

**Comment by davbotjohnson** [comment]
Sorry, I don't understand, do you mean like just clicking on them in the
menu or like starting them in the terminal or or something?

**Comment by george-hopkins** [comment]
I'm sorry I misread your question. Starting the applications in the terminal should be a good start. Are there any error messages?

**Comment by davbotjohnson** [comment]
I'm not sure how to run applications from the terminal, how do you do that?

**Comment by george-hopkins** [comment]
You can get to a terminal in two different ways:
* directly on the device: open Tools > Terminal
* over USB:
  - connect the device to your computer
  - start network: USB > Device > Ethernet Gadget
  - start telnet: Network > Telnet Server
  - open a terminal on your computer (enter `Win`+`R`,  `cmd` on Windows)
  - connect over telnet: `telnet 192.168.0.200`

Afterwards, enter the name of the application you like to start. In [nxmenu.cfg](https://github.com/george-hopkins/opentom/blob/master/src/opentom_skel/etc/nxmenu.cfg) you can see how the menu entries are mapped to commands (e.g. Dune 2 -> dune2.sh)
